pkg/handler/eval_test.go

```diff
@@ -171,6 +176,26 @@ func TestEvalFlag(t *testing.T) {
 		assert.NotNil(t, result.VariantID)
 	})
 
+	t.Run("test multiple segments with the first segment 0% rollout", func(t *testing.T) {
+		f := entity.GenFixtureFlag()
+		f.Segments = append(f.Segments, entity.GenFixtureSegment())
+		f.Segments[0].Constraints = []entity.Constraint{}
+		f.Segments[0].RolloutPercent = uint(0)
```

Comment:
The first segment says `0%` rollout for all entity (no constraints), thus the evaluation result should have `nil` variantID.